vm/x86/x86defs/src/lib.rs

```diff
@@ -121,6 +121,17 @@ pub struct SegmentRegister {
     pub attributes: SegmentAttributes,
 }
 
+impl Default for SegmentRegister {
+    fn default() -> Self {
+        Self {
+            base: 0,
+            limit: 0,
+            selector: 0,
+            attributes: SegmentAttributes(0),
+        }
+    }
+}
```

Comment:
Where is this being used?

Reply:
No longer necessary w/ the below discussed change to kill load_registers() for TDX. Removed.